lib/razorpay/refund.rb

```diff
@@ -19,5 +19,9 @@ def self.all(options = {})
     def self.fetch(id)
       request.fetch id
     end
+
+    def self.edit(id, options = {})
+      request.patch id, options
+    end
```

Comment:
Not an idiomatic way to implement this. Edit should work on an instance of refund obj. Pls see if we can rework on sdk arch when possible.

Reply:
yes this correct now ?
`para_attr = {
  "notes": {
    "notes_key_1": "Beam me up Scotty.",
    "notes_key_2": "Engage"
  }
}

Razorpay::Refund.fetch(refundId).edit(para_attr)`